scss/_lesson.scss

```diff
@@ -9,6 +9,20 @@
     h2, h3 {
         color: $header-color;
     }
+    .responsive-image {
```

Comment:
Can we have width and max-width at the same time?

Reply:
hey I had already added width and max-width. Do I need to change something or is there a better way of doing this?

Reply:
Actually I just confused because for img I usually saw only width or max-width (not both at the same time). Maybe I just don't have enough experience. I only saw similar usage both width and max-width when <picture> with several <source> elements inside was used.